src/main/java/seedu/address/model/person/Person.java

```diff
@@ -70,7 +70,8 @@ public boolean isSamePerson(Person otherPerson) {
         }
 
         return otherPerson != null
-                && otherPerson.getName().equals(getName());
+                && (otherPerson.getPhone().equals(getPhone())
+                || otherPerson.getEmail().equals(getEmail()));
```

Comment:
I think it is good that we check for both phone and email to ensure that there are no identical customer details with the same phone number or email. 👍